utils/fixtures/balancerV2Fixture.ts

```diff
@@ -0,0 +1,92 @@
+import { Signer } from "@ethersproject/abstract-signer";
+import { JsonRpcProvider, Web3Provider } from "@ethersproject/providers";
+import { ether } from "@utils/common";
+import DeployHelper from "@utils/deploys";
+import { Account } from "@utils/test/types";
+import { Address } from "@utils/types";
+
+import { StandardTokenMock, WETH9 } from "../contracts/index";
+import { BVault, WeightedPoolFactory } from "../contracts/balancerV2";
+import { BPoolV2__factory } from "../../typechain/factories/BPoolV2__factory";
+import { BigNumber } from "@ethersproject/bignumber";
+import { defaultAbiCoder } from "@ethersproject/abi";
```

Comment:
```suggestion
import { utils } from "@ethersproject/abi";
```

and below:

```typescript
    const initUserData = utils.defaultAbiCoder.encode(["uint256", "uint256[]"], [0, tokenAndAmounts.map(ta => ta.amount)]);
```

Reply:
Wouldn't I want to import this from `ethers` instead? Importing `utils` from the same `@ethersproject` library shouldn't have any meaningful effect. Am I missing something?

Reply:
Sorry that's my bad, you're absolutely right. I meant:

```suggestion
import { utils } from "ethers";
```